images/plbase/python-requirements.txt

```
@@ -32,7 +32,7 @@ scikit-learn==1.6.1
 scipy==1.15.2
 seaborn==0.13.2
 sphinx-markdown-builder==0.6.8
-sphinx==8.1.3
+sphinx==8.2.1
```

8.2.0 no longer supports Python 3.10.

```suggestion
sphinx==8.1.3
```